package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "start": "next dev",
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
```

Comment:
Any particular reason we're changing this?

Reply:
Could be critical top secret reasons regarding government security, or because a random code monkey felt like `npm start` was shorter than `npm run dev`. Tbh I never checked to see if the commands mapping to next commands was expected by next or vercel...

But yeah I can back this out, lmk if there's anything else you'd want changed before merging and I'll get it all in one commit